backend/erxes-api-shared/src/utils/trpc/index.ts

```
@@ -72,21 +71,23 @@ export const createTRPCContext =
     };
 
     if (trpcContext) {
       return await trpcContext(subdomain, context);
     }
 
-    return context;
+    return context as TContext & ITPRCContext;
   };
 
-export type ITRPCContext = Awaited<ReturnType<typeof createTRPCContext>>;
+export type ITRPCContext<TExtraContext = {}> = Awaited<
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)